### DISCUSSION TITLE
Заметки

### DISCUSSION THREAD

**Discussion by lunar-balance** [completely-burned/glowing-octo-shame-arma2#265] [General]
Ссылки, консольные команды, полезная информация, и прочее связанное с проектом

**Comment by lunar-balance** [comment]
Проверка изменений 

https://github.com/completely-burned/glowing-octo-shame-arma2/compare/master...dev

https://github.com/completely-burned/glowing-octo-shame-arma2/compare/master...pre

**Comment by completely-burned** [comment]
Не нужно вручную создавать релиз с именем например `v0.0.5.998*`, они создаются автоматически из одноимённых тэгов, что приводит к конфликтам.  
Чтобы сделать пуллреквест в не master ветку, то нужно перед этим в форке создать ветку из ветки нужного репозитория, а в форке автоматом обычно создаётся только главная ветка.

**Comment by completely-burned** [comment]
пуш  форка делать в форк, а слияние с оригиналом в веб интерфейсе github (пул реквест).
